Source/Mosa.Utility.Launcher/CheckOptions.cs

```diff
@@ -1,50 +1,39 @@
 ﻿// Copyright (c) MOSA Project. Licensed under the New BSD License.
 
-using Mosa.Compiler.Common.Configuration;
+using Mosa.Utility.Configuration;
 
 namespace Mosa.Utility.Launcher;
 
 public static class CheckOptions
 {
-	public static string Verify(Settings settings)
+	public static string Verify(MosaSettings settings)
 	{
-		var emulator = settings.GetValue("Emulator", string.Empty).ToLowerInvariant();
-		var imageformat = settings.GetValue("Image.Format", string.Empty).ToUpperInvariant();
-		var platform = settings.GetValue("Compiler.Platform", string.Empty);
+		settings.NormalizeSettings();
 
-		if (emulator == "qemu" && imageformat == "VDI")
+		if (settings.Platform != "x86" && settings.Platform != "x64")
 		{
-			return "QEMU does not support the VDI image format";
+			return $"Platform not supported: {settings.Platform}";
 		}
 
-		if (emulator == "bochs" && imageformat == "VDI")
+		switch (settings.Emulator)
 		{
-			return "Bochs does not support the VDI image format";
-		}
+			case "qemu" when settings.ImageFolder == "vdi":
```

Comment:
Can this QEMU+VDI check be removed? QEMU does actually support VDI.

Reply:
Removed.